tools/niminst/makefile.tmpl

```diff
@@ -12,146 +12,146 @@ binDir = ?{firstBinPath(c).toUnix}
 
 koch := $(shell sh -c 'test -s ../koch.nim && echo "yes"')
 ifeq ($(koch),yes)
-	binDir = ../bin
+  binDir = ../bin
```

Comment:
It's because I based this PR on #5830. If that gets merged first, this bit of the diff will go away.